packages/commons-ui-core/src/Section/index.js

```diff
@@ -1,16 +1,17 @@
 import { styled } from "@mui/material/styles";
+import { Container } from "@mui/material";
 
-const Section = styled("div")(({ theme }) => ({
+const ContainerStyled = styled(Container)(({ theme }) => ({
```

Comment:
```suggestion
const SectionRoot = styled(Container)(({ theme }) => ({
```

Sticking to MUI naming